examples/fit_with_bounds.py

```diff
@@ -18,21 +18,32 @@
 init_logging(level="debug")
 np.random.seed(15816)
 # {{{ helper function(s)
-class myfitclass (object):
-    def __init__(self):
+class myfitclass (nddata):
+    def __init__(self,*args,**kwargs):
+        # copied from fitdata
         self.expression = None
-    expression = (2+2)
-    def gen_from_expr(self,data, guesses={}):
+        if isinstance(args[0],nddata):
+            # move nddata attributes into the current instance
+            myattrs = normal_attrs(args[0])
+            for j in range(0,len(myattrs)):
+                self.__setattr__(myattrs[j],args[0].__getattribute__(myattrs[j]))
+        else:
+            nddata.__init__(self,*args,**kwargs)
+        return
+    @property
```

Comment:
what are these @ symbols for?

Reply:
They are decorators — good question.  If you look up a decorator tutorial, it will explain better than me.  Specifically here we are using the property decorator to create a property (which acts like an attribute but also does stuff behind the scenes when you set or get the value)